### ISSUE TITLE
Make the iOS app compatible with M1 Macs

### ISSUE THREAD

**Issue by KizzyCode** [simplex-chat/simplex-chat#455]
Would it be possible make the iOS app compatible with M1 Macs? Currently it states: "This app is not compatible with your Mac and cannot be downloaded."
![Screen Shot 2022-03-19 at 23 31 24](https://user-images.githubusercontent.com/13999009/159140723-2b576287-7470-43db-a738-fb4ebe51f27c.png)

**Comment by epoberezkin** [comment]
As far as I know you should be able to run some iOS apps on M1 macs - I’ve not tried it though yet - will do!.

**Comment by KizzyCode** [comment]
Yes, for most apps this works out of the box; however for Simplex and a few other Apps I get the error above (MacBook Pro 14", M1 Pro).
Thank you 😊

**Comment by PalmTamino** [comment]
Obsolete since there’s now a working desktop app. Close?

**Comment by epoberezkin** [comment]
we probably will be making a dedicated Mac app at some point (some time after we hire iOS engineer:), so I'd keep it open. Making it work on M1 would still be an important first step - it's not that hard tbh.
